trakt/types.go

```diff
@@ -65,6 +65,7 @@ type Show struct {
 	Overview      string   `json:"overview"`
 	Rating        float32  `json:"rating"`
 	Runtime       int      `json:"runtime"`
+	TagLine       string   `json:"tagline"`
```

Comment:
Trakt has `tagline` field as well?

Reply:
yes. i am pretty sure they take it from TMDB, since it the same.

```
$ curl -s -H @/home/antix/work/trakt-headers.txt https://api.trakt.tv/shows/supernatural?extended=full | jq .tagline
"Between darkness and deliverance"
```
```
$ curl -s 'https://api.themoviedb.org/3/tv/1622?api_key=29a551a65eef108dd01b46e27eb0554a&language=en' | jq .tagline
"Between darkness and deliverance"
```